cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -118,6 +118,11 @@ class ScannerCommand : CliktCommand(name = "scan", help = "Run existing copyrigh
         help = "Do not scan excluded projects or packages. Works only with the '--ort-file' parameter."
     ).flag()
 
+    private val createMissingArchives by option(
+        "--create-missing-archives",
+        help = "Creates missing archives for packages even if scan results are present in the storage."
```

Comment:
Use the imperative: s/Create/Creates/

And I'd say "Create archives for packages that have a stored scan result but no license archive yet."

Reply:
Thinking about this again, @mnonnenmacher should we make this a scanner configuration option instead of a CLI option, in our effort to remove configuration options from the CLI?

Reply:
Makes sense to me as it could still be passed on the command line if it's only required for a single run.

Reply:
Would you rather have it in the `ScannerConfiguration` or in the `FileArchiverConfiguration`? I cannot decide.

Reply:
I put it inside `FileArchiverConfiguration`.

Reply:
Sorry for replying (too) late... I would have put it in the `ScannerConfiguration`, as it's the scanner, not the archiver, who takes the configuration into account. I would have put it just below `val archive: FileArchiverConfiguration? = null`.